spec/lib/dttp/params/placement_assignment_spec.rb

```diff
@@ -133,6 +138,18 @@ module Params
               { "dfe_courselevel" => Dttp::Params::PlacementAssignment::COURSE_LEVEL_UG },
             )
           end
+
+          it "age range is 0 to 5" do
```

Comment:
super nitpick, but I don't like the grammatical nonsense here.  And in the spec below
```
      specify "age range is 0 to 5" do
```